File: qlty-cli/tests/lang/java/basic.in/cyclomatic/IterativeMethodsWithFilterAndInclude.java
import java.util.List;

class IterativeMethodsWithFilterAndInclude {
  public static void main(String[] args) {
    List<String> animals = List.of("dog", "cat", "bear", "tiger");

    animals.stream()
        .filter(animal -> animal.length() > 3)
        .forEach(animal -> System.out.println(animal));

    if (animals.contains("cat")) {
      System.out.println("Found a cat!");
    }
  }
}

File: qlty-cli/tests/lang/javascript/basic.in/cyclomatic/iterative_methods_with_filter_and_include.js
function foo() {
  let animals = ["dog", "cat", "bear", "tiger"];

  animals
    .filter((animal) => {
      return animal.length > 3;
    })
    .forEach((animal) => {
      console.log(animal);
    });

  if (animals.includes("cat")) {
    console.log("Found a cat!");
  }
}
